src/sqlfluff/rules/L006.py

```diff
@@ -51,7 +51,7 @@ def _missing_whitespace(seg, before=True):
         if seg.name.endswith("_bracket"):
             if seg.name.startswith("start_" if before else "end_"):
                 return False
-        if seg.is_meta:
+        if seg.is_meta:  # pragma: no cover
```

Comment:
This one worries me a little -- non-trivial rule code without tests are a pretty common source for bugs. Think we'd want to add a test for this? We could create a separate issue to look into this (potentially out of scope for this PR).

Reply:
Yeah I did spend some time trying to craft a test case for this but couldn't get figure it out.

Maybe we should add a `TODO` or `# pragma: no cover for now` option? But realistically is anyone ever going to clean those up so is there any value in that really? There's nothing to say this `pragma` can't be removed if this does cause a problem and a test is added!

As I say my aim here is to get 100% to make it reduce noise of slight decimal cases, rather than necessarily to get full coverage (though that would be great obviously!). I have added test cases where I could but think there is a question is to the value of knowing we have a gap and realistically never getting round to closing that gap, versus accepting that gap (at least for now) and using that to prevent other issues.

Adding the pragma won't make it worse than it is now IMHO - we have this risk of bugs in this code now as there is no test case. It's just accepting that and choosing not to highlight it each time.